R/ebp.R

```diff
@@ -84,14 +84,22 @@
 #' \code{\link[parallelMap]{parallelStart}}.
 #' @param custom_indicator a list of functions containing the indicators to be
 #' calculated additionally. Such functions must and must only depend on the
```

Comment:
Same as in direct, I think we can delete the must since it is not a must anymore, but must only is ok since it cannot have other arguments, right? Maybe, I would write .. depend on the target variable, the threshold and the pop_weight. The threshold was there before and it is not the same weight as the one in the direct, so maybe keep the more natural order since not everyone will use the population weights and I know its also true for the threshold. For me, it would just be more natural. What do you think?